client/layout/masterbar/notifications.jsx

```diff
@@ -54,14 +54,14 @@ class MasterbarItemNotifications extends Component {
 
 		// focus on main window if we just closed the notes panel
 		if ( this.props.isNotificationsOpen && ! isOpen ) {
-			this.getNotificationLinkDomNode().blur();
+			this.notificationLink.current.blur();
```

Comment:
Does this need a check on `current` being truthy before calling `.blur()`?

Reply:
`this.notificationLink.current` is guaranteed to be a non-null DOM node. The ref is attached in the render method:
```jsx
<div ref={ this.notificationLink } >
```
The `div` is always rendered, there is no conditional rendering that would sometimes render it and sometimes not. And at the time `componentWillReceiveProps` is called, the mounting and ref calling is guaranteed to be done.